components/StarshipCard/index.js

```diff
@@ -0,0 +1,24 @@
+import Image from "next/image";
+import styled from "styled-components";
+import Flex from "../Layout/Flex";
+
+const StyledImageWrapper = styled.section`
+  border-radius: 11px;
+  border: 1px solid var(--secondary-color);
+`;
```

Comment:
A `<section>` should always include a heading (h1, h2, etc...)

You can directly style the image like:

```js
const StyledImage = styled(Image)`
  border-radius: 11px;
  border: 1px solid var(--secondary-color);
`;
```

And skip the `<StyledImageWrapper>` in the component